common/autoconf/toolchain_windows.m4

```diff
@@ -25,7 +25,7 @@
 
 ################################################################################
 # The order of these defines the priority by which we try to find them.
-VALID_VS_VERSIONS="2010 2012 2013 2015 2017"
+VALID_VS_VERSIONS="2017 2019 2013 2015 2012 2010"
```

Comment:
I'm not sure whether it's appropriate to put 2017 and 2019 in front of the list - maybe for compatibilities sake we should add 2019 at the end?

Reply:
I completely agree with you but that is how it comes from 11u. I'm not sure if it's a good idea to make such changes as part of the backport. On my last backport to 8u @gnu-andrew made me to eliminate every signle whitespace of unwanted delta.

Reply:
Yes, let @gnu-andrew or @jerboaa deliver the judgement here.

Reply:
My understanding is that the order for different JDK releases may differ. Newer JDKs might want to build with the newest toolchain if possible. That's different for update releases. So if we backport this, versions for 2017 and 2019 should go to the end. See for example `8196108: Add build support for VS 2015/2017 ` which added this 4 years ago for 2015 and 2017.

TLDR; For compatibility we shall append.